### PR TITLE
feat(ui-components): add `Validate*` icons

### PR DESCRIPTION
**Pull request by johannes-kolbe**
PR adds validation icons, renames one
-   add `ValidateInfo` icon
-   add `ValidateError` icon
-   add `ValidateWarning` icon

### CODE REVIEW

**Review by 815are — DISMISSED** [review]
Checked:
1. color variables are valid;
2. change set presented
3. tested manually with storybook

@gpeuc @johannes-kolbe 
Small concern about icon rename - I think it is first time we rename existing icon, not sure if we are only who using such icon

**Review by Klaus-Keller — COMMENTED** [review]
See my comments about incompatible change.

File: packages/ui-components/src/components/Icons.tsx [hunk 1]
```diff
@@ -139,7 +139,10 @@ export enum UiIcons {
     Undo = 'Undo',
     Ui5 = 'Ui5',
     Validate = 'Validate',
-    ValidateBig = 'ValidateBig',
+    ValidateError = 'ValidateError',
+    ValidateInfo = 'ValidateInfo',
+    ValidateSuccess = 'ValidateSuccess',
+    ValidateWarning = 'ValidateWarning',
```
[the remark this answers is not in the document]
Comment (johannes-kolbe) [reply packages/ui-components/src/components/Icons.tsx:145]: Nevermind, seems like a code smell, reverted

**Review by Klaus-Keller — APPROVED** [review]
Thanks @johannes-kolbe, added icons look good and the change is compatible.

Approved.

**Pull request merged**